torch/csrc/distributed/rpc/tensorpipe_cuda.cpp

```diff
@@ -1,14 +1,18 @@
 #include <torch/csrc/distributed/rpc/tensorpipe_agent.h>
 #include <torch/csrc/distributed/rpc/tensorpipe_utils.h>
 
-#if defined(USE_TENSORPIPE) && !defined(USE_ROCM)
+#if defined(USE_TENSORPIPE)
 
 #include <c10/cuda/CUDACachingAllocator.h>
 #include <c10/cuda/CUDAGuard.h>
 #include <c10/cuda/CUDAStream.h>
 
 #include <tensorpipe/tensorpipe.h>
+#if defined(USE_ROCM)
+#include <tensorpipe/tensorpipe_hip.h>
+#else
 #include <tensorpipe/tensorpipe_cuda.h>
+#endif
```

Comment:
Why is this needed? If hipify-torch generates `tensorpipe/tensorpipe_hip.h`, it should take care of the header include hipification too?

Reply:
updated.